src/lib/core/ripple/ripple.ts

```diff
@@ -102,11 +107,11 @@ export class MdRipple implements OnChanges, OnDestroy {
 
   /** Ripple configuration from the directive's input values. */
   get rippleConfig(): RippleConfig {
-    return {
+    return extendObject({
       centered: this.centered,
       speedFactor: this.speedFactor,
       radius: this.radius,
       color: this.color
-    };
+    }, this._globalRippleConfig);
```

Comment:
How about adding a disabled field to RippleConfig? then you don't have to have a separate DI token